src/web/core/src/main/java/org/geoserver/web/util/LocalizationsFinder.java

```diff
@@ -38,6 +38,7 @@
             for (Resource resource :
                     resolver.getResources("classpath*:/GeoServerApplication_*.properties")) {
                 String name = resource.getFilename();
+<<<<<<< Updated upstream
```

Comment:
That presumably wasn't meant to get committed...

Reply:
Eh indeed... I'm doing programming, extra hard mode: write code while helping one kid with a "Lego techno" kit and checking the homework of my 16yo daughter on physics .....